app/views/users/show.html.erb

```diff
@@ -1,2 +1,36 @@
 <h1>Users#show</h1>
```

Comment:
- [ ] I would like to recommend you remove this `<h1> ` element and make sure that your user details page matches the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) as required for this project.  for example, the user photo is square, not rounded. Please fix it :+1: